### PR TITLE
charger logo added

### PR DESCRIPTION
**Pull request by ckgagan**
charger.dev logo added

### CODE REVIEW

**Comment by himalb001** [comment]
Thanks for the contribution @ckgagan! 

The PR is missing reference to the logo in the json and the markdown file. There's some more info on the steps in the readme file.

**Comment by ckgagan** [comment]
@himalb001 
I made the change in the  json and the markdown file. Does that look right?

File: img-buildkite-64.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 [
   {
+    "name": "charger",
+    "image": "img-buildkite-64/charger64.png",
+    "category": "Buildkite",
+    "aliases": []
+  },
+  }
```
Comment (himalb001) [inline img-buildkite-64.json:8]:
```suggestion
  {
```

Comment (ckgagan) [reply img-buildkite-64.json]: @himalb001 
json file error resolved.

**Review by himalb001 — APPROVED** [review]
Thanks for the contribution @ckgagan. The changes looks good to me. You should be able to use the emoji in the next few days!

**Pull request merged**